svg.js.d.ts

```diff
@@ -429,8 +434,14 @@ declare namespace svgjs {
         f: number;
         matrix: Matrix;
     }
+
+    interface Library {
+        Matrix: Matrix;
+    }
+
     export interface Matrix {
         (source: any): Matrix;
+        new(source: any): Matrix;

```

Comment:
Awesome.

I could cherry pick your commits and include them into the PR, if this is ok for you?